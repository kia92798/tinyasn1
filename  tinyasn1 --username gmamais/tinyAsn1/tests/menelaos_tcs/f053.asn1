F053 DEFINITIONS AUTOMATIC TAGS	
EXTENSIBILITY IMPLIED ::= BEGIN
ShortINTEGER ::= INTEGER (0..6)
Citizen ::= SET { country ENUMERATED {france, greece, italy, germany},
	    yearOfBirth	INTEGER,
	    married BOOLEAN,
	    nOfChildren ShortINTEGER}

	    GreekCitizen ::= Citizen (WITH COMPONENTS {...,
					country (greece)})    
	    aGreekCitizen GreekCitizen ::= {greece, 1980, TRUE, 4}

	    GreekCitizenTwoChildren ::= GreekCitizen (WITH COMPONENTS {...,
					nOfChildren (2)})
	    aGreekCitizenWithTwoChildren GreekCitizenTwoChildren ::= {greece, 1980, TRUE, 2}

	    EquivGreekCitizenTwoChildren ::= Citizen (WITH COMPONENTS {...,
					country(greece),
					nOfChildren (2)})


	    bGreekCitizenWithTwoChildren EquivGreekCitizenTwoChildren ::= {greece, 1980, TRUE, 2}

END

