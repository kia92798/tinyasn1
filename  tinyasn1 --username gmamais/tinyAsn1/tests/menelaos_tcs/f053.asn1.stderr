Error in line : 11. Expecting SEQUENCE variable
